src/View/Helper/BreadcrumbsHelper.php

```diff
@@ -0,0 +1,268 @@
+    }
+
+    /**
+     * Returns the crumbs list
+     *
+     * @return array
+     */
+    public function getCrumbs()
+    {
+        return $this->crumbs;
+    }
+
+    /**
+     * Renders the breadcrumbs trail
+     *
+     * @param array $attributes Array of attributes applied to the wrapper element
+     * @param array $separator Array of attributes for the `separator` template
+     * @return string The breadcrumbs trail
+     */
+    public function render(array $attributes = [], array $separator = [])
+    {
+        $crumbs = $this->crumbs;
+        $crumbsCount = count($crumbs);
+        $templater = $this->templater();
+
+        if (!empty($separator)) {
+            $separatorParams = [];
+            if (isset($separator['innerAttrs'])) {
+                $separatorParams['innerAttrs'] = $templater->formatAttributes($separator['innerAttrs']);
+                unset($separator['innerAttrs']);
+            }
+
+            if (isset($separator['separator'])) {
+                $separatorParams['separator'] = $separator['separator'];
+                unset($separator['separator']);
+            }
+
+            $separatorParams['attrs'] = $templater->formatAttributes($separator);
+        }
+
+        $crumbTrail = '';
+        foreach ($crumbs as $key => $crumb) {
+            $link = $this->prepareLink($crumb['link']);
+            $title = $crumb['title'];
+            $options = $crumb['options'];
+
+            $optionsLink = [];
+            if (isset($options['innerAttrs'])) {
+                $optionsLink = $options['innerAttrs'];
+                unset($options['innerAttrs']);
+            }
+
+            $template = 'item';
+            $templateParams = [
+                'attrs' => $templater->formatAttributes($options),
+                'innerAttrs' => $templater->formatAttributes($optionsLink),
+                'title' => $title,
+                'link' => $link,
+                'separator' => '',
+            ];
+
+            if (empty($link)) {
+                $template = 'itemWithoutLink';
+            }
+
+            if (isset($separatorParams) && $key !== ($crumbsCount - 1)) {
+                $templateParams['separator'] = $this->formatTemplate('separator', $separatorParams);
+            }
+
+            $crumbTrail .= $this->formatTemplate($template, $templateParams);
+        }
+
+        $crumbTrail = $this->formatTemplate('wrapper', [
+            'content' => $crumbTrail,
+            'attrs' => $templater->formatAttributes($attributes)
+        ]);
+
+        return $crumbTrail;
+    }
+
+    /**
+     * Prepare the URL for a specific `link` param of a crumb
+     *
+     * @param array|string|null $link If array, an array of Router url params
+     * If string, will be used as is
+     * If empty, will consider that there is no link
+     *
+     * @return null|string The URL of a crumb
+     */
+    protected function prepareLink($link)
+    {
+        if (is_string($link)) {
```

Comment:
Why is this check necessary? `UrlHelper::build()` should be called for string URLs too, so that webroot can be properly appended to the URLs (especially for cases where app is in a subfolder).